tests/system/action/meeting/test_import.py

```diff
@@ -0,0 +1,459 @@
+
+    def test_no_permission(self) -> None:
+        self.set_models(
+            {
+                "committee/1": {},
+                "meeting/1": {},
+                "motion/1": {},
+                "user/1": {
+                    "username": "admin",
+                    "organization_management_level": "can_manage_users",
+                },
+            }
+        )
+        response = self.request(
+            "meeting.import",
+            {
+                "committee_id": 1,
+                "meeting": {
+                    "meeting": [
+                        {
+                            "id": 1,
+                            "name": "Test",
+                            "description": "blablabla",
+                            "default_group_id": 1,
+                            "motions_default_amendment_workflow_id": 1,
+                            "motions_default_statute_amendment_workflow_id": 1,
+                            "motions_default_workflow_id": 1,
+                            "projector_countdown_default_time": 60,
+                            "projector_countdown_warning_time": 60,
+                            "reference_projector_id": 1,
+                        }
+                    ],
+                    "user": [
+                        {
+                            "id": 1,
+                            "password": "",
+                            "username": "admin",
+                            "group_$_ids": ["1"],
+                            "group_$1_ids": [1],
+                        },
+                        {
+                            "id": 2,
+                            "password": "",
+                            "username": "admin 1",
+                        },
+                    ],
+                    "group": [
+                        {"id": 1, "meeting_id": 1, "name": "testgroup", "user_ids": [1]}
+                    ],
+                    "motion_workflow": [
+                        {"id": 1, "meeting_id": 1, "name": "blup", "first_state_id": 1}
+                    ],
+                    "motion_state": [
+                        {
+                            "id": 1,
+                            "css_class": "line",
+                            "meeting_id": 1,
+                            "workflow_id": 1,
+                            "name": "test",
+                        }
+                    ],
+                    "projector": [{"id": 1, "meeting_id": 1}],
+                },
+            },
+        )
+        self.assert_status_code(response, 403)
+        assert (
+            "Missing CommitteeManagementLevel: can_manage" in response.json["message"]
+        )
+
+    def test_clean_blobs(self) -> None:
+        self.set_models(
+            {
+                "committee/1": {},
+                "meeting/1": {},
+                "motion/1": {},
+            }
+        )
+        response = self.request(
+            "meeting.import",
+            {
```

Comment:
For all tests: Can you reduce the payload to extract the important things? I would expect this to just have a mediafile, so it can be tested.

Reply:
Try to reduce the payload. Extracted the general part and build a create_request_data function.